GRASS community map (#169)
OSGeo/grass-website: content/about/map.md

---
title: "GRASS community map"

> date: 2018-12-29T11:02:05+06:00

Reviewer: ```suggestion
date: 2026-07-14T11:02:05+06:00
```

layout: "map"
---

### Add yourself

> To add yourself to the GRASS user map you need to follow [these instructions](https://github.com/OSGeo/grass-website/static/geojson/README.md).

Reviewer 2: ```suggestion
To add yourself to the GRASS community map, follow [these instructions](https://github.com/OSGeo/grass-website/static/geojson/README.md).
```

> To add yourself to the GRASS community map, follow [these instructions](https://github.com/OSGeo/grass-website/static/geojson/README.md).

Reviewer 3: ```suggestion
To add yourself to the GRASS community map, follow [these instructions](https://github.com/OSGeo/grass-website/blob/master/static/geojson/README.md).
```
